mmdet/models/losses/iou_loss.py

```diff
@@ -444,18 +466,21 @@ def forward(self,
 @MODELS.register_module()
 class DIoULoss(nn.Module):
 
-    def __init__(self, eps=1e-6, reduction='mean', loss_weight=1.0):
-        super(DIoULoss, self).__init__()
+    def __init__(self,
+                 eps: float = 1e-6,
+                 reduction: str = 'mean',
+                 loss_weight: float = 1.0):
+        super().__init__()
         self.eps = eps
         self.reduction = reduction
         self.loss_weight = loss_weight
 
     def forward(self,
-                pred,
-                target,
-                weight=None,
-                avg_factor=None,
-                reduction_override=None,
+                pred: Tensor,
+                target: Tensor,
+                weight: Optional[Tensor] = None,
+                avg_factor: Optional[int] = None,
+                reduction_override: Optional[str] = None,
                 **kwargs):
```

Comment:
Check whether add  `-> xx`  in all functions

Reply:
OK. I have checked the return value of all functions and modified the docstring.